api/client/src/main/java/org/projectnessie/client/NessieClientBuilder.java

```diff
@@ -398,6 +410,21 @@ public NessieClientBuilder fromConfig(Function<String, String> configuration) {
         withApiCompatibilityCheck(Boolean.parseBoolean(s));
       }
 
+      s = configuration.apply(CONF_NESSIE_COMMIT_MESSAGE);
+      if (s != null) {
+        withHttpHeader("Nessie-Commit-Message", s);
+      }
+      s = configuration.apply(CONF_NESSIE_COMMIT_AUTHORS);
+      if (s != null) {
+        withHttpHeader("Nessie-Commit-Authors", s);
```

Comment:
to clarify: I meant splitting the config value on `,` here, and setting it as multiple headers.